### DISCUSSION TITLE
Silent Install

### DISCUSSION THREAD

**Discussion by evittech** [keepassxreboot/keepassxc#7228] [Q&A] [answered]
Is there any documentation on the command line install options? I'd like to do a silent install w/ auto-start on W64 disabled. I've searched but not found anything as of yet.

**Comment by droidmonkey** [comment]
Not yet but you can see our custom properties here: https://github.com/keepassxreboot/keepassxc/blob/develop/share/windows/wix-template.xml#L109

**Comment by Zorziel** [comment, marked as answer]
@evittech, do you mean installing the MSI silently?  If so, you can use all [standard Windows Installer command line switches](https://docs.microsoft.com/en-us/windows/win32/msi/standard-installer-command-line-options).  

For example, to install version 2.6.6 completely silent, you would do:
```cmd
msiexec /i KeePassXC-2.6.6-Win64.msi /q
```

To install it with a basic UI/progress bar and also log to C:\temp.log with a verbose log, you would do: 
```cmd
msiexec /i KeePassXC-2.6.6-Win64.msi /qb /l*v C:\temp.log
```

To install it silently and disable launch after install and autostart but still log, you would do: 
```cmd
msiexec /i KeePassXC-2.6.6-Win64.msi /q LAUNCHAPPONEXIT=0 AUTOSTARTPROGRAM=0 /l*v C:\temp.log
```

Here's a screenshot of the Properties table from the MSI for 2.6.6 x64: 
![image](https://user-images.githubusercontent.com/46036567/155119531-6e0934ce-13be-478a-b156-8ba5ab2a3bd9.png)
